Fedora 20 / gcc 4.8.3 compilation is broken

```
make[2]: Entering directory `/home/pasha/work/ucx/build-dev/test/perf'
  CC       libperf.lo
/home/pasha/work/ucx/contrib/../test/perf/libperf.c: In function 'uct_perf_test_setup_endpoints':
/home/pasha/work/ucx/contrib/../test/perf/libperf.c:305:14: warning: variable 'group_index' set but not used [-Wunused-but-set-variable]
     unsigned group_index, group_size, i;
              ^
```

Nothing wrong with the code. I think gcc 4.8.3 is broken.

Workaround for this:

```
diff --git a/configure.ac b/configure.ac
index be1f032..4f4d924 100644
--- a/configure.ac
+++ b/configure.ac
@@ -90,7 +90,7 @@ m4_ifdef([AS_VAR_APPEND],
 #
 # Initialize CFLAGS
 #
-CFLAGS="-g -Wall -Werror $UCX_CFLAGS"
+CFLAGS="-g -Wall $UCX_CFLAGS"
 CXXFLAGS="$CFLAGS"
```

So, I suggest remove `-Werror` . Any other ideas ?

--- a/test/perf/libperf.c
+++ b/test/perf/libperf.c
@@ -302,7 +302,7 @@ static ucs_status_t ucx_perf_run_put_lat(uct_perf_context_t *perf)
 
 ucs_status_t uct_perf_test_setup_endpoints(uct_perf_context_t *perf)
 {
-    unsigned group_index, group_size, i;
+    unsigned group_size, i;
     uct_iface_addr_t *iface_addr;
     uct_ep_addr_t *ep_addr;
     uct_iface_attr_t iface_attr;
@@ -340,7 +340,6 @@ ucs_status_t uct_perf_test_setup_endpoints(uct_perf_context_t *perf)
 
     address = (uintptr_t)perf->super.recv_buffer;
 
-    group_index = rte_call(&perf->super, group_index);
     group_size  = rte_call(&perf->super, group_size);
 
     perf->peers = calloc(group_size, sizeof(*perf->peers));

